docs/source/concepts/architectural-model.rst

```diff
@@ -0,0 +1,284 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _connecting-ledger-model:
+
+Connecting the Ledger Model and the Ledger API
+##############################################
+
+This document connects the :ref:`DA ledger model <da-ledgers>` with the :ref:`ledger API services <ledger-api-services>` of the DAML platform.
+It explains how the ledger model captures the ledger API services of the DAML platform and what :ref:`guarantees <ledger-api-guarantees>` different :ref:`DAML ledger topologies <daml-ledger-topologies>` provide.
+The current version covers the following ledger API services:
+
+* :ref:`transaction service <transaction-service>`
+* :ref:`active contract service <active-contract-service>`
+
+It does not yet cover :ref:`submission services <ledger-api-submission-services>`, :ref:`utility services <ledger-api-utility-services>`, and :ref:`testing services <ledger-api-testing-services>`.
+
+The presentation assumes that you are familiar with the following concepts:
+
+* The :ref:`DA ledger model <da-ledgers>`
+  
+* The :ref:`ledger API services <ledger-api-services>`
+
+* The :ref:`DAML ledger topologies <daml-ledger-topologies>`
+
+   
+Multiple domains for the virtual shared ledger
+**********************************************
+
+In the DA ledger model, the virtual shared ledger consists of a :ref:`sequence of commits <ledger-structure>`.
```

Comment:
Surely we need to change the phrase "sequence of commits". Otherwise you get into trouble trying to explain multi-domain topologies where the virtual shared ledger is no longer linearly ordered. Maybe we should switch to a "DAG of commits". Otherwise you'd have to say that each domain is a virtual shared ledger with a broader term for multi-domain setups. Virtual Shared Multi-Ledger? blergh